pkg/apis/config/v1alpha1/resmgr/rdt/config.go

```diff
@@ -0,0 +1,29 @@
+// Copyright The NRI Plugins Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rdt
+
+// Config provides runtime configuration for class-based LLC cache control.
+// +k8s:deepcopy-gen=true
+type Config struct {
+	// Enable class based last level cache control. When enabled, policy
```

Comment:
Maybe _cache allocation and memory bandwidth control_ or so. RDT can also do L2 (on some platforms) and MBM

Reply:
Updated.